### ISSUE TITLE
I completed the model conversion and inference according to the tutorial, but the model performed poorly. There are some "warning" prompts during the model conversion that I cannot resolve, and I think they are the cause of the poor model performance.

### ISSUE THREAD

**Issue by YANSU7** [open-mmlab/mmdeploy#2042]
### Checklist

- [X] I have searched related issues but cannot get the expected help.
- [X] 2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/tree/main/docs/en/faq.md) but cannot get the expected help.
- [ ] 3. The bug has not been fixed in the latest version.

### Describe the bug

04/29 11:12:53 - mmengine - WARNING - Failed to search registry with scope "mmseg" in the "Codebases" registry tree. As a workaround, the current "Codebases" registry in "mmdeploy" is used to build instance. This may cause unexpected failure when running the built modules. Please check whether "mmseg" is a correct scope, or whether the registry is initialized.
04/29 11:12:54 - mmengine - WARNING - Failed to search registry with scope "mmseg" in the "mmseg_tasks" registry tree. As a workaround, the current "mmseg_tasks" registry in "mmdeploy" is used to build instance. This may cause unexpected failure when running the built modules. Please check whether "mmseg" is a correct scope, or whether the registry is initialized.
f:\experimentaldata\02-originaltraceability\c++loading\mmdeploy_mmseg\mmsegmentation\mmseg\models\builder.py:36: UserWarning: ``build_loss`` would be deprecated soon, please use ``mmseg.registry.MODELS.build()`` 
  warnings.warn('``build_loss`` would be deprecated soon, please use '
f:\experimentaldata\02-originaltraceability\c++loading\mmdeploy_mmseg\mmsegmentation\mmseg\models\losses\cross_entropy_loss.py:235: UserWarning: Default ``avg_non_ignore`` is False, if you would like to ignore the certain label and average loss over non-ignore labels, which is the same with PyTorch official cross_entropy, set ``avg_non_ignore=True``.
  warnings.warn(
Loads checkpoint by local backend from path: segformer_mit-b5_512x512_160k_ade20k_20210726_145235-94cedf59.pth
04/29 11:12:55 - mmengine - WARNING - DeprecationWarning: get_onnx_config will be deprecated in the future. 
04/29 11:12:55 - mmengine - INFO - Export PyTorch model to ONNX: work_dir/trt/resnet\end2end.onnx.
04/29 11:12:56 - mmengine - WARNING - Can not find torch._C._jit_pass_onnx_autograd_function_process, function rewrite will not be applied
04/29 11:12:56 - mmengine - WARNING - Can not find torch.nn.functional._scaled_dot_product_attention, function rewrite will not be applied
04/29 11:12:56 - mmengine - WARNING - Can not find torch._C._jit_pass_onnx_deduplicate_initializers, function rewrite will not be applied
D:\ProgramData\Anaconda3\envs\mmdeploy_mmseg\lib\site-packages\mmdeploy\codebase\mmseg\models\segmentors\base.py:38: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  img_shape = [int(val) for val in img_shape]
f:\experimentaldata\02-originaltraceability\c++loading\mmdeploy_mmseg\mmsegmentation\mmseg\models\utils\shape_convert.py:15: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert L == H * W, 'The seq_len doesn\'t match H, W'
04/29 11:14:19 - mmengine - INFO - Execute onnx optimize passes.
04/29 11:14:19 - mmengine - WARNING - Can not optimize model, please build torchscipt extension.
More details: https://github.com/open-mmlab/mmdeploy/tree/1.x/docs/en/experimental/onnx_optimizer.md
04/29 11:14:31 - mmengine - INFO - Successfully loaded tensorrt plugins from D:\ProgramData\Anaconda3\envs\mmdeploy_mmseg\lib\site-packages\mmdeploy\lib\mmdeploy_tensorrt_ops.dll

### Reproduction

from mmdeploy.apis import torch2onnx
from mmdeploy.apis.tensorrt import onnx2tensorrt
from mmdeploy.backend.sdk.export_info import export2SDK
import os

img = 'mmsegmentation/demo/demo.png'
work_dir = 'work_dir/trt/resnet'
save_file = 'end2end.onnx'
deploy_cfg = 'mmdeploy/configs/mmseg/segmentation_tensorrt_static-512x512.py'
model_cfg = 'segformer_mit-b5_8xb2-160k_ade20k-512x512.py'
model_checkpoint = 'segformer_mit-b5_512x512_160k_ade20k_20210726_145235-94cedf59.pth'
device = 'cuda'

# 1. convert model to IR(onnx)
torch2onnx(img, work_dir, save_file, deploy_cfg, model_cfg,
  model_checkpoint, device)

# 2. convert IR to tensorrt
onnx_model = os.path.join(work_dir, save_file)
save_file = 'end2end.engine'
model_id = 0
device = 'cuda'
onnx2tensorrt(work_dir, save_file, model_id, deploy_cfg, onnx_model, device)

# 3. extract pipeline info for sdk use (dump-info)
export2SDK(deploy_cfg, model_cfg, work_dir, pth=model_checkpoint, device=device)

### Environment

```Shell
CUDA：11.1
Cudnn：8.9
pytorch：1.9.0
mmdeploy-1.0.0rc3-windows-amd64-cuda11.1-tensorrt8.2.3.0
TensorRT-8.2.3.0.Windows10.x86_64.cuda-11.4.cudnn8.2
```

**Comment by RunningLeon** [comment]
@YANSU7 hi, the performance you referred to is `mAP`? If so, pls. double check the test pipeline in model config.
